framework/src/source/BaseTestSuite.bs

```diff
@@ -1526,7 +1809,9 @@ namespace rooibos
       try
         if type(node) = "roSGNode"
           if not rooibos.common.nodeContains(node, value)
-            msg = "Node doesn't have the '" + rooibos.common.asString(value, true) + "' value."
+            if msg = ""
+              msg = `expected "${rooibos.common.truncateString(rooibos.common.getTypeWithComponentWrapper(node, true))}" to contain child "${rooibos.common.truncateString(rooibos.common.getTypeWithComponentWrapper(value, true))}" by reference`
+            end if
             m.currentResult.fail(msg, m.currentAssertLineNumber)
```

Comment:
Triple check the intention of this assertion. It seems to look for a given child but the doc block makes it sound like something else.